src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markDone() {
+        this.isDone = true;
+    }
+    public void markNotDone() {
+        this.isDone = false;
+    }
```

Comment:
I like that your names representing methods are verbs and written in camelCase, good job following the coding standards.